examples/demo2/theme.rs

```diff
@@ -0,0 +1,138 @@
+#![allow(unused)]
+use palette::{named::LIGHTGRAY, white_point::B};
```

Comment:
Not sure how this ended up in here. I think I might have added the unused to keep some colors below around instead of losing them when changing which things are which colors. And then tab completed LIGHTGRAY instead of LIGHTGRAY etc. pulling in the wrong imports

Reply:
Removed